docs/refmans/gui/viselements/corelements/scenario_selector.md_template

```diff
@@ -77,6 +107,157 @@ application can then use the selected value.
+first field:
+<figure class="tp-center">
+  <img src="../scenario_selector-filter3-d.png" class="visible-dark"  width="80%"/>
+  <img src="../scenario_selector-filter3-l.png" class="visible-light" width="80%"/>
+  <figcaption>Filter criteria dialog</figcaption>
+</figure>
+
+After selecting a criterion, the 'Action' dropdown updates with comparison options specific to the
+selected criterion.<br/>
+The available comparisons depend on the type of the selected field:
+
+- Boolean: Options include "is" and "is not." The value field accepts True or False.
+- String: Options include "is," "is not," and "contains." The value field accepts a text
+  value.
+- Number: Options include "equals," "does not equal," "is less than," "is less than or equal to,"
+  "is greater than or equal to" and "is greater than." The value field accepts a numerical
+  value.
+- Date: Options include "is on," "is not on," "is before," "is on or before," "is on or after" and
+  "is after." The value field accepts a date value.
+
+To list all data nodes with labels containing the string "ina", the user would configure a filter as
+follows:
+<figure class="tp-center">
+  <img src="../scenario_selector-filter4-d.png" class="visible-dark"  width="80%"/>
+  <img src="../scenario_selector-filter4-l.png" class="visible-light" width="80%"/>
+  <figcaption>Create a new filter</figcaption>
+</figure>
+
+To add this filter to the filters list, the user must press the [MUI:Check] (Check) icon.<br/>
+Multiple filters can be added as needed.
+
+To close the filter configuration dialog, the user must click outside the dialog area.
+
+Once filters are added, they are applied immediately. The filtered scenario list appears as shown:
+<figure class="tp-center">
+  <img src="../scenario_selector-filter5-d.png" class="visible-dark"  width="80%"/>
+  <img src="../scenario_selector-filter5-l.png" class="visible-light" width="80%"/>
+  <figcaption>Filters applied</figcaption>
+</figure>
+
+You can see that only scenarios with matching label now appear in the list.<br/>
+The [MUI:FilterList] icon (Filter) in the control's tool bar has an overlay icon indicating that
+a filter with one criterion is currently applied.
+
+<h4>Removing a filter criteria</h4>
+
+To remove a filter from the list, the user would open the filters dialog ([MUI:FilterList] icon),
+locate the criterion that must be removed and delete it pressing the [MUI:Delete] icon (Delete).
+
+## Sorting
+
+By default, the Scenario Selector sorts scenarios in ascending order by their creation date. When
+applicable, scenarios are grouped by their respective Cycle.
+
+Users can customize the sorting order by defining custom sorting criteria, similar to the process of
+filtering. This allows users to create a prioritized sequence of sorting rules.
+
+Each sorting criterion specifies:
+
+- The scenario  property to sort by (e.g., label, creation date).
```

Comment:
double space between scenario and property (not that it matters in any way)

Reply:
👍